bigtable/CMakeLists.txt

```diff
@@ -47,6 +55,19 @@ include(${PROJECT_SOURCE_DIR}/cmake/EnableClangTidy.cmake)
 # We use abseil.io .
 set(BUILD_TESTING OFF)
 include(${PROJECT_SOURCE_DIR}/cmake/IncludeAbseil.cmake)
+# config options
+if (WIN32)
+  # /wd4005  macro-redefinition
+  # /wd4068  unknown pragma
+  # /wd4244  conversion from 'type1' to 'type2'
+  # /wd4267  conversion from 'size_t' to 'type2'
+  # /wd4800  force value to bool 'true' or 'false' (performance warning)
+  add_compile_options(/W3 /wd4005 /wd4068 /wd4244 /wd4267 /wd4800)
```

Comment:
These options look MSVC specific, but you are defining them for all WIN32 builds.  There are other compilers for Windows, GCC for example.  Please protect using MSVC.

Reply:
Done.